test/backend/wayland/conftest.py

```diff
@@ -2,7 +2,11 @@
 import os
 import textwrap
 
-from libqtile.backend.wayland.core import Core
+try:
```

Comment:
Very hacky!

The conftest files all get imported when pytest is initialised. I'm not installing wayland in ReadTheDocs so I need to stop any `ImportError`s due to missing files/modules. Installing Wayland on ReadTheDocs doesn't seem like a viable option (can only use apt sources for Ubuntu 18.04).

Reply:
Seems to be that all conftest files are parsed when pytest scans tests.

As for installing dependencies, no, I think you're ok. This is only an issue for running tests which most people aren't going to be doing.